Makefile

```diff
@@ -204,8 +192,15 @@ clean:
 	rm -rf build/
 
 check-env:
-ifndef IDF_PATH
-	$(error IDF_PATH is not set, if you want to use the Toitware fork execute "export IDF_PATH=`pwd`/third_party/esp-idf" (see README.md))
+ifeq ("$(wildcard $(IDF_PATH)/components/mbedtls/mbedtls/LICENSE)","")
+ifeq ("$(IDF_PATH)", "$(CURDIR)/third_party/esp-idf")
+	$(error mbedtls sources are missing. Did you initialize the submodules?)
+else
+	$(error Invalid IDF_PATH. Missing mbedtls sources.)
+endif
+endif
+ifneq ("$(IDF_PATH)", "$(CURDIR)/third_party/esp-idf")
+	$(info -- Not using Toitware ESP-IDF)
```

Comment:
Should we call this "Toitware's ESP-IDF fork" consistently?

Reply:
done.